app/webpacker/src/stylesheets/_help_popup.scss

```diff
@@ -0,0 +1,32 @@
+#help_popup {
+  background-color: #ffffff;
+  border: 1px #333 solid;
+  border-radius: 0.5em;
+  padding: 0.3em 0.5em;
+  width: 180px;
+  position: fixed;
+  right: 50px;
+  bottom: 50px;
+  box-shadow: 0 3px 10px rgb(0 0 0 / 0.2);
+  display: none;
+
+  img {
+    width: 24px;
+    position: relative;
+    top: 8px;
+    padding-right: 2px;
+  }
+
+  p {
+    font-family: "GDS Transport", arial, sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    font-size: 14px;
+    line-height: 1.25;
+    margin: 0.75em;
+  }
+
+  .bold {
```

Comment:
This feels a bit redundant? would a `<strong>` tag do with the added benefit its semantic?